app/schema/api/v4/schema.rb

```diff
@@ -141,7 +141,10 @@ def user_me_response
 
     def facility_teleconsultations_response
       { type: :object,
-        properties: { teleconsultation_phone_number: { type: [:string, 'null'] } },
+        properties: { teleconsultation_phone_number: { type: [:string, 'null'] },
+                      teleconsultation_phone_numbers: { type: :array,
+                                                        items: { type: :object,
+                                                                 properties: { phone_number: { type: :string}}}}},
```

Comment:
The `teleconsultation_phone_numbers` key should be required, right? It will always be present. Same for the `phone_number` property inside. When no telemed info is available, the API should return an empty array.

`teleconsultation_phone_numbers: []`